srcpkgs/root/template

```diff
@@ -0,0 +1,96 @@
+# Template file for 'root'
+pkgname=root
+version=6.26.02
+revision=1
+# Only i686 and x86_64 seem to be officially supported
+archs="i686* x86_64*"
+build_style=cmake
+configure_args="-Dgnuinstall=ON -Druntime_cxxmodules=OFF
+ -Dclad=OFF $(vopt_if fortran '-Dfortran=ON')
+ -DXROOTD_XrdCl_LIBRARY=/usr/lib/libXrdCl.so.3
+ -DCMAKE_INSTALL_LIBDIR=/usr/lib/root -Dr=ON
+ -DCMAKE_INSTALL_SYSCONFDIR='../etc/root'
+ $(vopt_if root7 '-Droot7=ON -DCMAKE_CXX_STANDARD=17 -Dwebgui=ON')
+ -DCMAKE_INSTALL_PYTHONDIR=${py3_sitelib/'usr/'}"
+hostmakedepends="gcc-fortran"
+makedepends="libX11-devel libXpm-devel libXft-devel libXext-devel
+ openssl-devel libpng-devel readline-devel tbb-devel liblz4-devel
+ pcre-devel xxHash-devel libzstd-devel liblzma-devel libvdt-devel
+ libxml2-devel python3-devel python3-numpy R-cran-Rcpp R-cran-RInside
+ xrootd"
+depends="gcc"
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)
+	configure_args+=" -Dssl=OFF"
+	;;
+esac
+short_desc="C++ framework for data processing created by CERN"
+maintainer="Ben Jargowsky <benjar63@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://root.cern"
+distfiles="https://root.cern/download/root_v${version}.source.tar.gz"
+checksum=7ba96772271a726079506c5bf629c3ceb21bf0682567ed6145be30606d7cd9bb
+build_options="fortran root7"
+build_options_default="fortran"
+python_version=3
+shlib_provides="libCore.so libTree.so libRIO.so libMatrix.so libThread.so
+ libTMVA.so libMathCore.so"
+
+post_install() {
+	rm ${DESTDIR}/usr/share/root/fonts/LICENSE
+	rm ${DESTDIR}/usr/share/root/fonts/arialbd.ttf
```

Comment:
`rm -v ${DESTDIR}/usr/share/root/fonts/[a-z]*.ttf` maybe.

Reply:
That's much cleaner. I made it `rm -v ${DESTDIR}/usr/share/root/fonts/[a-mt-z]*.ttf` to avoid removing one GPL font (symbol.ttf).